<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>UsersWindow</class>
    <widget class="QMainWindow" name="UsersWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1041</width>
                <height>533</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="windowTitle">
<<<<<<< HEAD
            <string>Residents List</string>
=======
            <string>List of residents of the town</string>
>>>>>>> e9ef449c1c2c11323532d65843d3a6e2a4b976d6
        </property>
        <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QTextEdit" name="searchTextBox">
                <property name="geometry">
                    <rect>
                        <x>580</x>
                        <y>480</y>
                        <width>381</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Arial</family>
                    </font>
                </property>
                <property name="layoutDirection">
                    <enum>Qt::RightToLeft</enum>
                </property>
                <property name="inputMethodHints">
                    <set>Qt::ImhNone</set>
                </property>
                <property name="placeholderText">
                    <string/>
                </property>
            </widget>
            <widget class="QPushButton" name="addResidentButton">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>480</y>
                        <width>171</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Arial</family>
                    </font>
                </property>
                <property name="text">
<<<<<<< HEAD
                    <string>Add New Resident</string>
=======
                    <string>New resident registration</string>
>>>>>>> e9ef449c1c2c11323532d65843d3a6e2a4b976d6
                </property>
            </widget>
            <widget class="QLabel" name="label_9">
                <property name="geometry">
                    <rect>
                        <x>960</x>
                        <y>480</y>
                        <width>71</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Arial</family>
                    </font>
                </property>
                <property name="text">
                    <string>Search</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                    <rect>
                        <x>310</x>
                        <y>550</y>
                        <width>251</width>
                        <height>26</height>
                    </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QRadioButton" name="radioLName">
                            <property name="text">
<<<<<<< HEAD
                                <string>Last Name</string>
=======
                                <string>Last name</string>
>>>>>>> e9ef449c1c2c11323532d65843d3a6e2a4b976d6
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QRadioButton" name="radioPlateNum">
                            <property name="text">
<<<<<<< HEAD
                                <string>Plate Number</string>
=======
                                <string>License plate</string>
>>>>>>> e9ef449c1c2c11323532d65843d3a6e2a4b976d6
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
            <widget class="QTableWidget" name="tableWidget">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>10</y>
                        <width>1011</width>
                        <height>461</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>Arial</family>
                    </font>
                </property>
                <property name="layoutDirection">
                    <enum>Qt::RightToLeft</enum>
                </property>
                <property name="alternatingRowColors">
                    <bool>false</bool>
                </property>
                <property name="rowCount">
                    <number>15</number>
                </property>
                <property name="columnCount">
                    <number>3</number>
                </property>
                <attribute name="horizontalHeaderCascadingSectionResizes">
                    <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderCascadingSectionResizes">
                    <bool>false</bool>
                </attribute>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <column/>
                <column/>
                <column/>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
